Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -1,16 +1,27 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Globalization;
+using System.IO;
 
 namespace BootCamp.Chapter
 {
     public class ContactsCenter
     {
-        private readonly List<Person> _people;
+        private readonly string _peopleFile;
```

Comment:
This is bad. Why should a contacts center keep _peopleFile if all they need is people? Before it was good.

Reply:
So please revert to what it was with just that TryParse pattern applied.

Reply:
I did delete the List part from my first commit 
I have now this : 
```
 private readonly List<Person> _people;

        public ContactsCenter(string peopleFile)
        {
            if (!File.Exists(peopleFile))
            {
                throw new FileNotFoundException("File is made up");
            }

            var contents = File.ReadAllLines(peopleFile);

            if (contents.Length == 0)
            {
                throw new InvalidOperationException("File cannot be empty");
            }

            for (int i = 1; i < contents.Length; i++)
            {
                var isValid = Person.TryParse(contents[i], out Person person);

                if (isValid)
                {
                    _people.Add(person);
                }

            }
       }
```

and will tomorrow think how I can initialize the _people list.